src/useGet.test.tsx

```diff
@@ -945,9 +994,10 @@ describe("useGet hook", () => {
         </RestfulProvider>,
       );
 
-      await wait(() => expect(children).toBeCalledTimes(3));
-      expect(children.mock.calls[2][0].loading).toEqual(false);
-      expect(children.mock.calls[2][0].data).toEqual({ id: 1 });
+      // A mystery... this is called an extra time
```

Comment:
Even if this not a big issue, a bit of investigation over there could be nice (at least to understand ^^)